cmd/gosky/debug.go

```diff
@@ -231,3 +242,152 @@ var debugStreamCmd = &cli.Command{
 		return nil
 	},
 }
+
+// TODO: WIP - turns out to be more complicated than i initially thought
```

Comment:
feels like this is almost a separate tool, though don't want to proliferate those too much. maybe do move code for this specific command to a separate file at least? it is pretty big